samples/matter/basic/README.rst

```diff
@@ -0,0 +1,129 @@
+.. _matter_basic_sample:
+
+Matter: Basic
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample presents a minimal implementation that enables a device to be commissioned into a :ref:`Matter <ug_matter>` network.
+The main goal of this sample is to provide a basis for developing your own application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp
+
+For remote testing scenarios, if you want to commission the device and :ref:`control it remotely <matter_basic_sample_network_mode>` through a Thread network, you also need the following:
+
+* A Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>`.
+
+.. note:: |matter_gn_required_note|
+
+Overview
+********
+The sample automatically starts BLE advertising, getting ready for commissioning into a Thread network.
+It uses LED to show the state of its connection, and a button to allow factory reset when needed.
+
+Testing requires a Matter controller, which can be configured either on PC or mobile (for remote testing in a network).
+Both methods can be enabled after :ref:`building and running the sample <matter_basic_remote_control>`.
+
+.. _matter_basic_sample_network_mode:
+
+Remote testing in a network
+===========================
+
+.. matter_basic_sample_remote_testing_start
```

Comment:
Those labels are only needed if you want to automaticaly copy part of one document into another. In your case, they can be removed imho.